client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,44 @@
+import Box from '@material-ui/core/Box';
+import { User } from '../../interface/User';
+import LogoHeader from '../LogoHeader/LogoHeader';
+import NavbarTitle from '../NavbarTitle/NavbarTitle';
+import AuthMenu from '../AuthMenu/AuthMenu';
+
+interface Props {
+  loggedInUser: User;
+}
+
+const Navbar = ({ loggedInUser }: Props): JSX.Element => {
+  const titlesArray = loggedInUser.isSitter
+    ? [
+        { title: 'notifications', link: '/notifications', withBadge: true },
+        { title: 'my jobs', link: '/myJobs', withBadge: true },
```

Comment:
```suggestion
        { title: 'my jobs', link: '/my-jobs', withBadge: true },
```
Please use hyphenated case for route names. Ditto throughout!